config/eslintrc.yaml

```diff
@@ -22,32 +23,80 @@ globals:
 
 parserOptions:
   ecmaFeatures:
-    # Remove after migration
+    # Remove after migration.
     legacyDecorators: true
 
 rules:
   prettier/prettier: error
 
+  # Enforce import order.
+  import/order:
```

Comment:
Formatting import order is a good start, however if we really want to solve the import disaster we have I suggest that we treat the `Well-known modules` folders from the `Import Statement Order` (L427) section as standalone packages. 

We can achieve that by using [webpack aliases](https://webpack.js.org/configuration/resolve/#resolvealias) combined with [eslint-import-resolver-webpack](https://www.npmjs.com/package/eslint-import-resolver-webpack). With such approach we dont need to account for the depth of `..`.

As an example, instead of `../../../../components/...` we will use `components/...` or even better `@ttn-lw/components/...` or similar that will facilitate later changes to actually make these packages as standalone modules. Same applies to `console` , `oauth`, `lib`.

Imo, it makes sense to implement this functionality in this PR.